modules/ai/preferences.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace Elementor\Modules\Ai;
+
+use Elementor\User;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Preferences {
+
+	const ENABLE_AI = 'elementor_enable_ai';
+
+	/**
+	 * Register actions and hooks.
+	 *
+	 * @return void
+	 */
+	public function register() {
+		add_action( 'profile_personal_options', function ( \WP_User $user ) {
+			$this->add_personal_options_settings( $user );
+		} );
+
+		add_action( 'personal_options_update', function ( $user_id ) {
+			$this->update_personal_options_settings( $user_id );
+		} );
+	}
+
+	/**
+	 * Determine if AI features are enabled for a user.
+	 *
+	 * @param int $user_id - User ID.
+	 *
+	 * @return bool
+	 */
+	public static function is_ai_enabled( $user_id ) {
+		return (bool) User::get_user_option_with_default( static::ENABLE_AI, $user_id, true );
+	}
+
+	/**
+	 * Add settings to the "Personal Options".
+	 *
+	 * @param \WP_User $user - User object.
+	 *
+	 * @return void
+	 */
+	protected function add_personal_options_settings( \WP_User $user ) {
+		if ( ! $this->has_permissions_to_edit_user( $user->ID ) ) {
+			return;
+		}
+
+		$ai_value = User::get_user_option_with_default( static::ENABLE_AI, $user->ID, '1' );
```

Comment:
You can escape the id once and reuse:

```suggestion
		$ai_value = User::get_user_option_with_default( static::ENABLE_AI, $user->ID, '1' );
		
		$escaped_id = esc_attr( static::ENABLE_AI );
```